setup/mail-postfix.sh

```diff
@@ -215,6 +215,83 @@ tools/editconf.py /etc/default/postgrey \
 tools/editconf.py /etc/postfix/main.cf \
 	message_size_limit=134217728
 
+# ### MTA-STS - SMTP Mail Transfer Agent Strict Transport Security - SETUP
+# See: https://github.com/mail-in-a-box/mailinabox/pull/1556
+# create the MTA-STS policy
+mkdir -p /var/lib/mailinabox/
+cat > /var/lib/mailinabox/mta-sts.txt << EOF
+version: STSv1
+mode: enforce
+mx: \$PRIMARY_HOSTNAME
```

Comment:
$PRIMARY_HOSTNAME shouldn't be escaped here;  This results in "$PRIMARY_HOSTNAME" being added to mta-sts.txt instead of the domain name

Reply:
Fixed https://github.com/mail-in-a-box/mailinabox/pull/1556/commits/15a827d1ec721e7d2b790909235a699fb7e2b404